inc/cleantalk-public-integrations.php

```diff
@@ -1714,6 +1714,8 @@ function apbct_form__contactForm7__tesSpam__before_validate($result = null, $_ta
         if ( ! empty($invalid_fields) && is_array($invalid_fields) ) {
             $apbct->validation_error = $invalid_fields[key($invalid_fields)]['reason'];
             apbct_form__contactForm7__testSpam(false);
+        } else {
+            apbct_form__contactForm7__testSpam(false);
```

Comment:
This code will be executed independent of the `if` condition. Please move this out from to simplify.